src/main/java/org/altbeacon/beacon/BeaconManager.java

```diff
@@ -476,6 +493,24 @@ public void setBackgroundMode(boolean backgroundMode) {
             }
         }
     }
+    public void setEnableScheduledScanJobs(boolean enabled) {
+        this.mScheduledScanJobsEnabled = enabled;
```

Comment:
since we are prefixing members with `m` per Android style, I agree that `this` adds nothing.